pkg/model/selector.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2021 CGI France
+//
+// This file is part of PIMO.
+//
+// PIMO is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// PIMO is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with PIMO.  If not, see <http://www.gnu.org/licenses/>.
+
+package model
+
+import "github.com/cgi-fr/pimo/pkg/selector"
+
+// Dictionary is a Map with string as key and Entry as value
+type Dictionary = selector.Dictionary
```

Comment:
```suggestion
type Dictionary selector.Dictionary
```

Reply:
This change will create two types for Dictionary (model.Dictionary and selector.Dictionary) that might confuse unit tests. The best solution might be to use `selector.` everywhere it need to be, and delete this type alias.

Reply:
Okay, this is too impactful for no clearly identified betterment ..
I revert the split of model package into selector package. This will be for another pull request dedicated to minimize the size of the model package.